lntest/itest/lnd_test.go

```diff
@@ -4488,16 +4488,6 @@ func testSingleHopSendToRoute(net *lntest.NetworkHarness, t *harnessTest) {
 		t.Fatalf("unable to create pay reqs: %v", err)
 	}
 
-	// We'll wait for all parties to recognize the new channels within the
-	// network.
-	ctxt, _ = context.WithTimeout(ctxb, defaultTimeout)
-	err = net.Bob.WaitForNetworkChannelOpen(ctxt, chanPointAlice)
-	if err != nil {
-		t.Fatalf("alice didn't advertise her channel in time: %v", err)
-	}
-
-	time.Sleep(time.Millisecond * 50)
```

Comment:
Add comment to commit msg explaining why it is not need. Are the channels already recognized at this point?

Reply:
done